src/socketio.quiver.js

```diff
@@ -602,18 +602,27 @@ QuiverSocialProvider.prototype.connect_ = function(server) {
     }
   }.bind(this);
 
+  var getMaxRetries = function() {
```

Comment:
I don't see the stylistic appeal of having this in a separate function.  I would probably move it into the `"connect_error"` listener, maybe even as a ternary like `var maxRetries = this.finishLogin_ ? 2 : 6;`